extra/debian/control

```diff
@@ -67,6 +68,7 @@ Depends: ${shlibs:Depends},
          python-lxml,
          python-lzma,
          python-opencv,
+         python-pysnmp4,
          python-serial,
          snmp,
```

Comment:
I almost forgot: Can you please remove the dependency on `snmp` now that `stbt-power.sh` doesn't need it.